Library/Homebrew/cmd/tap-info.rb

```diff
@@ -37,7 +37,7 @@ def print_tap_info(taps)
     else
       taps.each_with_index do |tap, i|
         puts unless i == 0
-        info = "#{tap}: "
+        info = "[#{tap.get_priority.to_s.rjust(2, '0')}] #{tap}: "
```

Comment:
Priority should probably be a separate line. Also, `tap-info` should probably display the core formulae and the priority there too for completeness.